### PR TITLE
Update Lockheed Martin

### PR DESCRIPTION
**Pull request by Friendseeker**
"We're sorry this job post no longer exists" for the two removed links.

The last remaining link available says the location is Fort Worth, Texas

Screen Shot 2022-10-05 at 8 00 45 PM

### CODE REVIEW

**Review by ewang2002 — APPROVED** [review]
Looks good.

If a company originally had 2 internships listed here but one of them closes, then redo the entry so that 
- the company name is hyperlinked so that clicking on it goes to the internship application link 
- the description just contains the job title and any other relevant information (no link needed here since the company only have one internship, thus no ambiguity). 
- the location is updated as needed

Basically, redo the row so that it looks like any other company with only one internship listed here. Thanks!
